### PR TITLE
Plasmeme eyes no longer get roasted when welding

### PR DESCRIPTION
**Pull request by Moltijoe**
Adds a snowflake check to welding tools that lets people wearing a plasmeme helmet weld without blinding themselves.
Doesn't help with flashes or other forms of blinding, just welding

Now plasmeme engineers don't get screwed over for simply existing

:cl:  
tweak: Plasmeme can now weld again
/:cl:

### CODE REVIEW

**Comment by knightshade0** [comment]
I LOVE YOU!

File: code/game/objects/items/tools/weldingtool.dm [hunk 1]
```diff
@@ -242,13 +242,21 @@
 /obj/item/weldingtool/tool_start_check(mob/living/user, amount=0)
 	. = tool_use_check(user, amount)
 	if(. && user)
+		if(ishuman(user))
+			var/mob/living/carbon/human/H
+			if(istype(H.head,/obj/item/clothing/head/helmet/space/plasmaman))
```
Comment (Simplehorror) [inline code/game/objects/items/tools/weldingtool.dm:247]: What the fuck?

Comment (Moltijoe) [reply code/game/objects/items/tools/weldingtool.dm:247]: ah, forgot to assign user to H, ty

**Comment by Cartlord** [comment]
why not just make some plasmaman helmets include toggle-able welding screens, or make them actually use welding goggles?

**Comment by Marmio64** [comment]
use welding goggles

**Comment by knightshade0** [comment]
Plasmamen's helmets are supposed to do that shit *for* them. It's just bugged for the experimental welder right now.

**Comment by Moltijoe** [comment]
> Plasmamen's helmets are supposed to do that shit _for_ them. It's just bugged for the experimental welder right now.

it didn't work for any welding tool when i tried it at first

**Comment by knightshade0** [comment]
I might be mistaken. I could have sworn it worked fine for the non-experimental ones.

**Comment by Moltijoe** [comment]
> use welding goggles

engineers don't have the liberty of using welding goggles if they need to weld, they generally need the engineering goggles at all time, that's where plasmeme get used most too

**Comment by Moltijoe** [comment]
> I might be mistaken. I could have sworn it worked fine for the non-experimental ones.

nope, tried with a few different ones, none worked

**Comment by Simplehorror** [comment]
make a component or something i beg

**Comment by Moltijoe** [comment]
> make a component or something i beg

making a component would vastly overcomplicate this and would offer no tangible benefit
the only difference is that i would check for the component rather than the specific helmet

**Review by JamieD1 — APPROVED** [review]
Eh fine.

**Pull request merged**